### ISSUE TITLE
Get block with name from a variable

### ISSUE THREAD

**Issue by ciadch** [smarty-php/smarty#221]
Hello there !

Is it possible to get a {block} with name issue fromw a vairable ?

```php
{assign var="name" value="block_name"}
{block name=$name}{/block}
```

Thanks !

**Comment by ciadch** [comment]
Sorry it work perfectly like thant 
```php
{assign var="name" value="block_name"}
{block name="$name"}{/block}
``

**Comment by uwetews** [comment]
Variable name attribute of {block} tags are not supported. 

```php
{assign var="name" value="block_name"}
{block name="$name"}{/block}
```
Must have worked only by chance in the Smarty version you have used.  Which one?

Why you need to use a variable name? Looks like a wrong approach.

**Comment by ciadch** [comment]
What I need with variable name is to load a specific block from a template file with a lot of block. For example : I have a template file which contains blocks with different <input>. Some of them need to be loaded by Ajax dynamically. It'as why I need to use name as variable. Maybe you have a better approach for that ? 
Thanks

**Comment by uwetews** [comment]
Variable {block} tag names are now supported.
The change is in the master branch and will later be included in 3.1.30
